sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/models/SignedIdentifiersWrapper.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables.implementation.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+import java.util.List;
+
+/** A wrapper around List&lt;SignedIdentifier&gt; which provides top-level metadata for serialization. */
+@JacksonXmlRootElement(localName = "SignedIdentifiers")
+public final class SignedIdentifiersWrapper {
```

Comment:
Is there an issue to get the code generator working correctly to generate this type?

Reply:
Honestly, I don't know how to get it to do that, so I added the class manually and was looking to add a transform later (once I learn enough about autorest).